gnd/src/main/res/drawable-anydpi/ic_area_download.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think for this operation the "+" icon would be more appropriate (i.e. add an offline area); I'd expect "download" to trigger/start a download when pressed, whereas here it opens a UI to select an area for download. @jacobmclaws please cmiiw.

Reply:
Sounds reasonable. I've switched it to ic_add_light. 

Technically we should remove the download icon at this point...but do you think we'll use it later? I can sort of envision using it in the area selector, but we could also just use text, in which case we should scrap the download icon.